libraries/compute-utils/function-views/src/rich-function-view.ts

```diff
@@ -174,7 +198,9 @@ export class RichFunctionView extends FunctionView {
   private getStandardButtons(): HTMLElement[] {
     const runButton = this.getRunButton();
     const runButtonWrapper = ui.div([runButton]);
-    ui.tooltip.bind(runButtonWrapper, () => runButton.disabled ? (this.isRunning.value ? 'Computations are in progress' : 'Some inputs are invalid') : '');
+    ui.tooltip.bind(
+      runButtonWrapper,
+      () => runButton.disabled ? (this.isRunning.value ? 'Computations are in progress' : this.getValidationMessage()) : '');
```

Comment:
I am sure it is better to keep validation messages inside of tooltips of invalid inputs, too. They will be visible on any hover over input - this will increase user-friendliness

Reply:
I think, we will get old waiting for this :) 
We could keep it as it is now and introduce tooltips later